_episodes/11-snakemake-intro.md

```diff
@@ -89,7 +91,7 @@ number of words in the text file.
 
 We can do the same thing for a different book:
 
-```bash
+```
 $ python wordcount.py books/abyss.txt abyss.dat
 $ head -5 abyss.dat
 ```
```

Comment:
Do we need a `{: .language-bash}` here?

Reply:
Yes-- it's at the end of the block, not the beginning. This ` ```bash ` tag was severely confusing the lesson-check script.